core/src/main/java/com/linecorp/armeria/common/logging/RequestContextExporter.java

```diff
@@ -423,6 +427,10 @@ private static String getAuthority(RequestContext ctx, HttpHeaders headers) {
         return null;
     }
 
+    private static void exportId(Map<String, String> out, RequestContext ctx) {
+        putBuiltInProperty(out, REQ_ID, ctx.id().text());
```

Comment:
Please address about this `ctx.id().text()`

`RequestId` class also has `shortText()` for human-readable. I think later one is better, but this can make people confuse if two id is nearly same.

Reply:
I'd lean towards using full, not human readable. This is often used for searching logs based on the ID so precision is nice.